suite-native/module-send/src/components/OutputsReviewFooter.tsx

```diff
@@ -145,6 +172,27 @@ export const OutputsReviewFooter = ({
             }
 
             setTxid(sentTxid);
+        } else if (isRejected(sendResponse)) {
+            showAlert({
+                icon: 'warningCircle',
+                title: <Translation id="moduleSend.review.toasts.sendTxnFailed" />,
+                description:
+                    sendResponse.payload?.metadata.payload.error ?? sendResponse.error.message,
+                primaryButtonTitle: <Translation id="generic.buttons.close" />,
+                primaryButtonVariant: 'redBold',
+                onPressPrimaryButton: () => {
+                    hideAlert();
+                    dispatch(cleanupSendFormThunk({ accountKey, shouldDeleteDraft: true }));
+                    navigation.dispatch(
+                        navigateToAccountDetail({
```

Comment:
Are we sure that we want to navigation back to the account detail from here? What about navigation the user back to the send form, so he can try it again? Does @shenkys know that we will show an extra alert?

Reply:
d05da19b76